utils/emulator_launcher.py

```diff
@@ -493,7 +493,7 @@ def backdoor_get_from_device(self, variable_name, *_, **__):
         """
         Return the string of a value on a device from lewis.
         :param variable_name: name of the variable
-        :return: the variables value, as a string
+        :return: the variables value
         """
         # backdoor_command returns a list of bytes and join takes str so convert them here
```

Comment:
I was a little confused by this comment - does `str` here refer to the `str(variable_name)` in this command, or does it refer to the `str` that has been removed from `backdoor_command` above?